packages/manager/apps/key-management-service/src/public/translations/key-management-service/terminate/Messages_fr_FR.json

```
@@ -3,5 +3,6 @@
   "key_management_service_terminate_confirm": "Résilier",
   "key_management_service_terminate_error": "Une erreur est survenue. {{error}}",
   "key_management_service_terminate_heading": "Résilier mon KMS",
-  "key_management_service_terminate_description": "Veuillez entrer le mot « TERMINATE » pour confirmer la résiliation de votre KMS"
+  "key_management_service_terminate_description": "Veuillez entrer le mot « TERMINATE » pour confirmer la résiliation de votre KMS",
+  "key_management_service_terminate_success_banner": "Votre demande de résiliation pour {{ServiceName}} a bien été prise en compte, vous allez recevoir un mail pour valider celle-ci."
```

The message is a bit weird 
```suggestion
  "key_management_service_terminate_success_banner": "Votre demande de résiliation de votre KMS {{ServiceName}} a été prise en compte. Un e-mail contenant la procédure vous a été envoyé."
```